src/panels/logbook/ha-logbook.ts

```diff
@@ -38,7 +38,9 @@ class HaLogbook extends LitElement {
     const oldHass = changedProps.get("hass") as HomeAssistant | undefined;
     const languageChanged =
       oldHass === undefined || oldHass.language !== this.hass.language;
-    return changedProps.has("entries") || languageChanged;
+    const rtlChanged =
+      oldHass === undefined || computeRTL(oldHass) !== computeRTL(this.hass);
```

Comment:
This is not needed, we already update if the language changes and `rtl` can not change without a lang change.

Reply:
But lang can change and RTL may not. If this is being over-efficient let me know and I'll happily throw it out

Reply:
It isn't efficient at all, to check if you have to set it you have already calculated the value twice, and then you are calculating it again.

If you would just set it if the language changes you would calculate it once instead of atleast 2 or 3 times.

Reply:
I removed the rtl change check altogether. It's not 100% efficient but fine. This is how it was there to begin with.

Reply:
If you want to optimize it, change line 45, and check if the language is changed before setting rtl.